mods/species/vox/datum/species.dm

```diff
@@ -151,6 +152,9 @@
 		H.equip_to_slot_or_del(new /obj/item/storage/box/vox(H), BP_R_HAND)
 		H.set_internals(backpack)
 
+// Ideally this would all be on bodytype, but pressure is handled per-mob currently.
+var/global/list/vox_current_pressure_toggle = list()
```

Comment:
not a huge fan of this. maybe i'll rework it to use an extension added on species/bodytype set, but for now it works. though, noting `ref()` and `\ref[]` can be pretty expensive due to string ops.